pkg/shared/src/StockMap.ts

```diff
@@ -85,6 +140,11 @@ export class StockMap implements VolumeStockMap {
 	// #endregion Map interface
 
 	// #region Additional methods
+	/**
+	 * Aggregates the entries into the stock.
+	 * @param entries
+	 * @returns
+	 */
 	aggragate(entries: Iterable<VolumeStockInput>) {
```

Comment:
I know it's not part of this PR, but it makes sense to fix this typo now

`aggragate` → `aggregate`